.circleci/config.yml

```diff
@@ -0,0 +1,32 @@
+version: 2.1
+orbs:
+  node: circleci/node@4.0.1
+
+jobs:
+  lint:
+    executor:
+      name: node/default
+      tag: 'current'
+    steps:
+      - checkout
+      - node/install-packages
+      - run:
+          command: |
+            npm run build:prod
```

Comment:
Should this be `build`? `build:prod` minifies the code.

Reply:
Happy to adjust! I used `build:prod` to ensure we were testing code that's a close to our prod deployed code as possible, but I agree it isn't really necessary in this case (and skipping minifying will speed things up a bit). Thanks!